packages/blitz/src/cli/commands/next/start.ts

```diff
@@ -17,12 +17,15 @@ const start: CliCommand = async () => {
     },
   )
 
+  const extraArgs = nextArgs["_"].filter((arg) => arg !== "start")
+
   const config: ServerConfig = {
     rootFolder: process.cwd(),
     port: nextArgs["--port"],
     hostname: nextArgs["--hostname"],
     inspect: nextArgs["--inspect"],
     env: process.env.NODE_ENV === "production" ? "prod" : "dev",
+    extraArgs,
```

Comment:
hey thank you! Been wanting this for awhile.

Few things

- we need to make sure extraArgs is not also including ones we explicitly use (like `--inspect`)
- we should remove our use of `--hostname` and `--port` while we are here, they are legacy and no longer needed

Reply:
hi @julianklumpers, are you able to make these changes?

Reply:
yes, sorry for my late reply. Will try to implement this very soon.